dev/Dockerfile.dev

```diff
@@ -1,6 +1,6 @@
 FROM python:3.10.3-slim-buster
 
-RUN apt-get update && apt-get install --yes make git curl ncat vim bash-completion mime-support gcc libpq-dev
+RUN apt-get update && apt-get install --yes make git curl ncat vim bash-completion mime-support gcc libpq-dev libmagic-dev
```

Comment:
And also here: Instead of `libmagic-dev` you should use `libmagic1`, see doc of `python-magic`